Makefile

```diff
@@ -28,7 +28,8 @@ update_db : import_directory import_db auth_models.json flush_db link_locations
 	python manage.py loaddata auth_models.json
 	python manage.py make_materialized_views --recreate
 	python manage.py update_countries_plus
-	python manage.py make_search_index --recreate
+	python manage.py rebuild_index --noinput
```

Comment:
do we need a command to build the schema?

also, it would seem like these search index commands maybe belong under a different PHONY target?